composer.json

```diff
@@ -109,6 +109,8 @@
         "drupal/components": "^1",
         "drupal/composer_deploy": "^1.4",
         "drupal/conditional_fields": "1.x-dev",
+        "drupal/config_filter": "^2.2",
+        "drupal/config_ignore": "2.x-dev@dev",
```

Comment:
What does the `@dev` suffix do? I dont see it on any other packages in this file, and we use a lot of dev packages.

Reply:
This is how it was written on drupal.org page. Not sure to be honest.

Reply:
Lets omit it please.

Reply:
Done

Reply:
Done